### ISSUE TITLE
unifi-controller port for UAP/USW/USG to inform controller is missing

### ISSUE THREAD

**Issue by gr3k** [truenas/apps#368]
During deploying unifi-controller app it is only possibility to setup below ports:
WebUI HTTPS Port
WebUI HTTP Port
Portal HTTPS Port
Portal HTTP Port

From documentation:https://github.com/goofball222/unifi
This container exposes the following ports (see: https://help.ubnt.com/hc/en-us/articles/218506997-UniFi-Ports-Used):

3478/udp (port used for STUN connection)
6789/tcp (port used for throughput measurement from Android/iOS app)
8080/tcp (port for UAP/USW/USG to inform controller)
8443/tcp (port for controller GUI / API)
8880/tcp (port for HTTP portal redirect)
8843/tcp (port for HTTPS portal redirect)
10001/udp (port used for UBNT discovery broadcasts - Local LAN/L2/same subnet only)

At least this port is needed :
8080/tcp port for UAP/USW/USG to inform controller)

without it ,I cannot adopt my current AP to this controller.
Naming schema could be also this same as from github page...

**Comment by stavros-k** [comment]
Hey, the inform port is not really missing, but the naming is indeed hard to follow.
And its also how unifi names those ports
https://github.com/goofball222/unifi/blob/a6be6dd39bf20b8e82ee8c722fa078d733d57946/archive/8.2/official/root/usr/lib/unifi/system.properties.default#L5-L16

I've gone ahead and updated the labels for the ports hoping it will make it clearer.
Also added the measurment port as well.

The UDP ports for discovery, won't work unless host network is enabled, which will bind the ports automatically anyway, without explicitly defining them.
https://github.com/truenas/apps/pull/372
